projects/layout/components/navigation/hint-aside.directive.ts

```diff
@@ -23,7 +25,7 @@ export class TuiHintAsideDirective {
         computed(() =>
             this.aside.expanded() || this.dropdown
                 ? ''
-                : () => this.el.textContent?.trim(),
+                : () => this.domSanitizer.bypassSecurityTrustHtml(this.el.innerHTML),
```

Comment:
I'm not sure I like this. Maybe we should make it possible to pass a template instead like we do in HintOverflow?

Reply:
fixed